fw-child/template/post-meta.php

```diff
@@ -1,57 +1,59 @@
-<div class="news-meta-item mb-3 mb-lg-4">
-	<h6 class="mb-3 all-caps text-blue-100"><?php _e ( 'Date', 'cdc' ); ?></h6>
-	
-	<?php 
-		
-		the_time ( __( 'F j, Y', 'cdc' ) );
-	
+<div class="news-meta-block p-3 border border-gray-600">
+	<div class="news-meta-item mb-3 mb-lg-4">
+		<h5 class="mb-3 all-caps text-blue-100"><?php _e ( 'Date', 'cdc' ); ?></h5>
```

Comment:
@sebastian-ew I would not recommend not using headings here. You can instead use <span> for labels.

Reply:
@MahieddineAk I made the change from heading tag to span. Thanks!